__tests__/app.test.js

```diff
@@ -56,4 +57,39 @@ describe('app.js', () => {
             })
         })
     })
+
+    describe('GET /api/articles/:article_id', () => {
+        test('200: responds with the article object requested', () => {
+            return request(app).get('/api/articles/1')
+            .expect(200)
+            .then(({body}) => {
+                const {article} = body
+                expect(article).toMatchObject(
+                    {
+                        title: "Living in the shadow of a great man",
```

Comment:
Have the article_id property as well so you we can super confirm it is the same article as in the path